### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by bellom**
Kindly Review !

### CODE REVIEW

File: src/component/Display.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Display(props) {
+  const { result } = props;
+
+  return (
+    <div>
+      <h3>Display Component</h3>
+      <p>{result}</p>
```
Comment (SunnyAsar) [inline src/component/Display.js:9]: Hi, let us set defaultProps to make sure there is `0` displayed when there is no prop.
and also set propTypes to string

File: src/component/Button.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+function Button(props) {
+  const { buttonName } = props;
+  return (
+    <div>
+      <button>{buttonName}</button>
+    </div>
+  );
+}
+
```
Comment (SunnyAsar) [inline src/component/Button.js:14]: let's set propType to string for the buttonName here too.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hey @bellom  👋 
Project looking good so far.

#### For the finish:
- let up just update the display and button components.
The display needs a default value, the display and button need to have propTypes also set.

**Please request another review when these changes are in place**

**Comment by bellom** [comment]
**Note:** Modern React, we can always initialize our "state" without constructor.
e.g state = {firstName: ""}

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @bellom ,

Great job, but we need to set up [propTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) to validate the props in Display and Button components:
* Display's result prop should be a string with a default value of '0'
* Button's name prop should be a string and it should be required

**Please add it and submit another review request.**

_If you have any questions please contact me on Slack._

**Review by BalintMendli — APPROVED** [review]
Hi @bellom ,

Looking good now.

### 👍 

Let's move to the next one.

**Pull request merged**